local.c

```diff
@@ -1480,6 +1480,21 @@ static unsigned int is_global_attr(struct iio_channel *chn, const char *attr)
 
 	len = ptr - attr;
 
+	// Check for matching global differential attr, like "voltage-voltage"
+	dashptr = strchr(attr, '-');
+	if (dashptr && dashptr > attr && dashptr < ptr) {
+		unsigned int len1 = dashptr - attr,
+					 len2 = ptr - dashptr - 1;
+		const char*  iddashptr = strchr(chn->id, '-');
+		if (iddashptr && strlen(iddashptr+1) > len2 &&
```

Comment:
Please add spaces before and after the '+' sign here and in the lines that follow.